maintainers/maintainer-list.nix

```diff
@@ -3393,6 +3393,12 @@
       fingerprint = "74B1 F67D 8E43 A94A 7554  0768 9CCC E364 02CB 49A6";
     }];
   };
+  haskie = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add haskie``.